.github/libs/GitUtils.js

```diff
@@ -1,5 +1,6 @@
 const _ = require('underscore');
 const {spawn} = require('child_process');
+const sanitizeStringForJSONParse = require('../../src/libs/sanitizeStringForJSONParse');
```

Comment:
Since you have an ES6 default export in `sanitizeStringForJSONParse` I would expect you to have to do this to import it into CJS (via babel):

```suggestion
const sanitizeStringForJSONParse = require('../../src/libs/sanitizeStringForJSONParse').default;
```